src/generators/dart.ts

```diff
@@ -90,54 +104,50 @@ export function _toDart(requests: Request[], warnings: Warnings = []): string {
     s += "\n";
   }
 
-  // TODO: Uri() can accept a params dict
   const queryIsRepresentable =
-    request.urls[0].queryList &&
-    request.urls[0].queryDict &&
-    Object.values(request.urls[0].queryDict).every((v) => !Array.isArray(v));
+    request.urls[0].queryList && request.urls[0].queryDict; /*&&
+    Object.values(request.urls[0].queryDict).every((v) => !Array.isArray(v))*/
   if (queryIsRepresentable && request.urls[0].queryList) {
-    // TODO: dict won't work with repeated keys
-    s += "  var params = {\n";
-    for (const [paramName, rawValue] of request.urls[0].queryList) {
+    s += "  final params = {\n";
+    for (const [
+      _,
+      [paramName, rawValue],
+    ] of request.urls[0].queryDict!.entries()) {
       const paramValue = repr(rawValue ?? new Word(), imports);
```

Comment:
```suggestion
      const paramValue = reprWords(rawValue, imports);
```

the `?? new Word()` isn't necessary anymore, it used to be that rawValue here could be null.